test/test_autograd.py

```diff
@@ -975,6 +975,7 @@ def test_no_requires_grad_inplace(self):
         with self.assertRaises(RuntimeError):
             b.add_(5)
 
+    @unittest.skipIf(TEST_WITH_ROCM, "test doesn't currently work on the ROCm stack")
```

Comment:
If you defined `skipIfRocm`, why don't you use it?

Reply:
@jithunnair-amd comment/fix, thanks!

Reply:
We had been able to make do without defining it until I had to define it for test_gesv and some other tests. I will make another PR to clean up all uses of TEST_WITH_ROCM and replace it with the skipIfRocm annotation. I didn't want to clutter up this PR. Let me know if that doesn't work for you.